mahara-autoscale-cache/azuredeploy.json

```diff
@@ -0,0 +1,734 @@
+            },
+            "type": "string"
+        },
+        "caCertThumbprint": {
+            "defaultValue": "",
+            "metadata": {
+                "description": "Thumbprint of your stored CA cert. This value can be obtained from keyvault.sh output if you used the script to store your CA cert in your Key Vault. This parameter is ignored if the keyVaultResourceId parameter is blank."
+            },
+            "type": "string"
+        },
+        "storageAccountType": {
+            "defaultValue": "Standard_LRS",
+            "allowedValues": [
+                "Standard_LRS",
+                "Standard_GRS",
+                "Standard_ZRS"
+            ],
+            "metadata": {
+                "description": "Storage Account type"
+            },
+            "type": "string"
+        },
+        "searchType": {
+            "defaultValue": "none",
+            "allowedValues": [
+                "none",
+                "elastic"
+            ],
+            "metadata": {
+                "description": "options of mahara global search"
+            },
+            "type": "string"
+        },
+        "elasticVmSku": {
+            "defaultValue": "Standard_DS2_v2",
+            "metadata": {
+                "description": "VM size for the elastic search nodes"
+            },
+            "type": "string"
+        },
+        "vNetAddressSpace": {
+            "defaultValue": "172.31.0.0",
+            "metadata": {
+                "description": "Address range for the Mahara virtual network - presumed /16 - further subneting during vnet creation"
+            },
+            "type": "string"
+        },
+        "gatewaySubnet": {
+            "allowedValues": [
+                "GatewaySubnet",
+ 		"MaharaGatewaySubnet",
+		"MyMaharaGatewaySubnet"
+            ],
+            "defaultValue": "GatewaySubnet",
+            "metadata": {
+                "description": "name for Virtual network gateway subnet"
+            },
+            "type": "string"
+        },
+        "gatewayType": {
+            "allowedValues": [
+                "Vpn",
+                "ER"
+            ],
+            "defaultValue": "Vpn",
+            "metadata": {
+                "description": "Virtual network gateway type"
+            },
+            "type": "string"
+        },
+        "vpnType": {
+            "allowedValues": [
+                "RouteBased",
+                "PolicyBased"
+            ],
+            "defaultValue": "RouteBased",
+            "metadata": {
+                "description": "Virtual network gateway vpn type"
+            },
+            "type": "string"
+        }
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2016-09-01",
+            "name": "pid-738e3eec-68d4-4667-8377-c05c77c21f1b",
```

Comment:
@rgardler - is this correct?

Reply:
No, this is not correct. This has been copied from a different template and should be removed from here. Thanks.